src/test/java/words/WordsTest.java

```diff
@@ -0,0 +1,14 @@
+package words;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WordsTest {
+
+    @Test
+    void select() {
+
+        assertEquals((Words.matches("\\b[A-Z]\\S+|\\b[A-Z]", "Word cAT DATA A site Java tExt")), "Word DATA A Java", "Word DATA A Java");
```

Comment:
`\\b[A-Z]\\S+|\\b[A-Z]` can be simplified: `+` means [1..) and `*` means [0..), so `OR` is not required if there's `*`

Reply:
fixed =)